be/src/storage/column_in_predicate.cpp

```diff
@@ -115,6 +115,17 @@ class ColumnInPredicate : public ColumnPredicate {
         return Status::OK();
     }
 
+    Status seek_inverted_index(const std::string& column_name, InvertedIndexIterator* iterator,
+                               roaring::Roaring* row_bitmap) const override {
+        InvertedIndexQueryType query_type = InvertedIndexQueryType::EQUAL_QUERY;
+        for (auto value : _values) {
+            roaring::Roaring indices;
+            RETURN_IF_ERROR(iterator->read_from_inverted_index(column_name, &value, query_type, &indices));
+            *row_bitmap |= indices;
```

Comment:
It seems to be wrong for IN predicate

Reply:
I think `in (a,b,c)` is equivalent to `a or b or c`, but the better way to implement this is to use `MultiTermQuery`, however, this change will affect the uniformity of the interface parameters, so I want to keep the interface as simple as possible in the early stages.